DataConnectors/Templates/Connector_CEF_template.json

```diff
@@ -110,5 +110,23 @@
             "title": "4. Secure your machine ",
             "description": "Make sure to configure the machine's security according to your organization's security policy\n\n\n[Learn more >](https://aka.ms/SecureCEF)"
         }
-    ]
+    ],
+    "metadata": {
+        "id": "Unique Identifier (GUID) used to identify dependencies and content from solutions or community.",
+        "version": "Optional: Version of the content. Default and recommended format is numeric (e.g. 1, 1.0, 1.0.0, 1.0.0.0), following ARM template best practices.",
+        "kind": "dataConnector",
+        "source": {
+            "kind": "source type of the content. Value must be one of these : localWorkspace | community | solution | sourceRepository",
+            "name": "Name of the content source. The repo name, solution name, LA workspace name etc."
+        },
+        "author": {
+            "name": "Name of the author. For localWorkspace it is automatically the workspace user"
+        },
+        "support": {
+            "tier": "Type of support for content item: microsoft | developer | community",

```

Comment:
Need to capitalize these values or is that managed at UX layer?

Reply:
These values are from the enum as mentioned in [contentMetadataSchema](https://msazure.visualstudio.com/One/_git/Sentinel-AzMP-Solutions?path=%2Fschema%2FcontentMetadata.schema.json)